src/routes.ts

```diff
@@ -16,15 +15,14 @@ import Signout from '@/views/auth/Signout.vue'
 import { DigitalCredentialRoutes } from '@/resources/DigitalCredentialRoutes'
 import { Routes } from '@/enums'
 import { FilingNames } from '@bcrs-shared-components/enums'
+import Dashboard from './views/Dashboard.vue'
 
 export default [
   {
+    // Return to BRD if the URL is hit directly
     path: '/',
```

Comment:
An example of this route is https://dev.business.bcregistry.gov.bc.ca/CP1049523/, right?

Why not let the default/fallback route handle this? (see also my next comment)

Reply:
Looks good. Resolving.